src/calculate-cost.ts

```
@@ -221,4 +208,4 @@ if (import.meta.vitest != null) {
 			});
 		});
 	});
-}
+}
```

ESLint

[error] Newline required at end of file but not found.

(style/eol-last)